src/engines/julia/update_rules/nonlinear_sampling.jl

```diff
@@ -48,20 +47,6 @@ function ruleSPNonlinearSIn1MN(g::Function,
     return Message(variateType(dims), Function, log_pdf = (z)->logPdf(msg_out.dist, g(z)))
 end
 
-function ruleSPNonlinearSOutNGX(g::Function,
```

Comment:
I suppose this method is no longer needed because of the new inheritance rules you introduced recently